Tasks/XamarinAndroid/Get-JavaDevelopmentKitPath.ps1

```diff
@@ -0,0 +1,96 @@
+function Get-JavaDevelopmentKitPath{
```

Comment:
The logic is similar to how the capabilities are detected in the vsts-agent: https://github.com/Microsoft/vsts-agent/blob/master/src/Misc/layoutbin/powershell/Add-JavaCapabilities.ps1